src/store/holochain.js

```diff
@@ -88,6 +88,15 @@ export default {
           commit('holoInitialized', { anonymous: true })
           dispatch('loadHoloAppInfo')
           dispatch('loadHostInfo')
+
+          const urlParams = new URLSearchParams(window.location.search)
+          if (urlParams.has('signin')) {
+            dispatch('holoSignin')
+          } else if (urlParams.has('signup')) {
+            dispatch('holoSignup')
+          } else {
+            console.log('No instructions to login provided in uri, proceeding as anonymous user.')
+          }
```

Comment:
this seems like a weird place to put this. the `connected` event is emitted every time we connect to a host, but we don't want to call signin every time we connect to a host, we want to call it when the page loads. I think it makes more sense to base this off of what I did in #129 where you add a watcher to `ElementalChat.vue` that checks for
```js
isHoloHosted() && this.holoStatus === 'ready' && this.isHoloAnonymous === true
```
I think you also want to add something that removes the search string once we have logged in:
```js
if (isHoloHosted() && this.holoStatus === 'ready' && this.isHoloAnonymous === false) {
  window.location.search = ''
}
```

Reply:
lol... that was silly!! I see I got a bit sidetracked and didn't do the removal.  Thanks for the keen eye! - Also, I was honestly looking for the the simplest place to quickly push through the update for the QA testing - but I do agree that it is not as optimized as it could be.  Your suggestion is more optimized though - updating now.